build.sh

```diff
@@ -0,0 +1,2 @@
+rm -rf dist
+sbt -Dsbt.parser.simple=true clean 'set publishMavenStyle in Global := false' 'set every publishTo := Some(Resolver.file("local-repo", file("dist"))(Patterns(Seq("[organisation]/[module]/[revision]/[module]-[revision](-[classifier]).ivy"),Seq("[organisation]/[module]/[revision]/[module]-[revision](-[classifier]).[ext]"),true)))' '+ publish'
```

Comment:
`'set publishMavenStyle in Global := false'` should be removed, since we will publish to Maven.

Reply:
Actually this can be as simple as `sbt -Dsbt.parser.simple=true clean 'set every publishTo := Some(Resolver.file("local-repo", file("dist")))' '+ publish'`

Reply:
Updated. Still wondering about `.asc`.